CHANGELOG

```diff
@@ -1,3 +1,8 @@
+## [0.6.0] - 2022-08-25
+
+- Added ability to configure a queue to send batches of messages
+  to a handling function rather than processing them individually
+
```

Comment:
Please use the [Keep a changelog](https://keepachangelog.com/en/1.0.0/) convention:
```suggestion
## [Unreleased]

### Added

- Added ability to configure a queue to send batches of messages
  to a handling function rather than processing them individually

```

Reply:
Corrected, also added a `CONTRIBUTING.md` and copied over the parts from the readme along with notes on the changelog format so I won't forget next time